airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -32,6 +33,19 @@
 __installed_version__ = importlib.metadata.version("pipelines")
 
 
+def display_welcome_message() -> None:
```

Comment:
Can we add the CI suffix to the printed message :D

Reply:
Done!